src/main/java/com/glodblock/github/common/block/BaseBlockContainer.java

```diff
@@ -0,0 +1,17 @@
+package com.glodblock.github.common.block;
+
+import java.util.List;
+import net.minecraft.block.BlockContainer;
+import net.minecraft.block.material.Material;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+
+public abstract class BaseBlockContainer extends BlockContainer {
```

Comment:
this base class is completely useless

Reply:
i need this to addInfomations

Reply:
you can add tooltip at `BaseItemBlockContainer` directly